test/README

```diff
@@ -76,12 +76,13 @@ You can set these environment variables to configure the test suite:
   TEST_OS    The OS to run the tests in.  Currently supported values: (vm-create)
                 "fedora-23"
                 "fedora-24"
+                "fedora-25"
                 "fedora-testing"
                 "fedora-atomic"
                 "centos-7"
                 "rhel-7"
                 "debian-unstable"
-             "fedora-24" is the default (testinfra.py, testlib.sh)
+             "fedora-25" is the default (testinfra.py, testlib.sh)
```

Comment:
If that is so, it should be marked as such in [testinfra.py:49](https://github.com/cockpit-project/cockpit/pull/5018/files#diff-6eef19f6e12cab1b25ceb2f3d8765caaR49)